rt_bene_model_training/dataset_manager.py

```
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+import cv2
+import numpy as np
+from os import listdir
+
+import json
+import csv
+from tqdm import tqdm
+import itertools
+
+
+def read_rgb_image(img_path, size, flip):
+    assert type(size) is tuple, "size parameter must be a tuple, (96, 96) for instance"
+    img = cv2.imread(img_path, cv2.IMREAD_COLOR)
+    if img is None:
+        print("ERROR: can't read " + img_path)
```

Raise an Exception rather than printing this.